src/jit/inline.cpp

```diff
@@ -578,8 +443,9 @@ void InlineContext::DumpData(Compiler* compiler, int indent)
     if (m_Parent == nullptr)
     {
         // Root method... cons up a policy so we can display the name
-        InlinePolicy* policy = InlinePolicy::GetPolicy(compiler, true);
-        printf("\nInlines [%u] into \"%s\" [%s]\n", compiler->fgInlinedCount, calleeName, policy->GetName());
+        InlinePolicy*   policy   = InlinePolicy::GetPolicy(compiler, true);
+        InlineStrategy* strategy = compiler->m_inlineStrategy;
```

Comment:
Isn't it the same as "m_Strategy"?
It's your call, but I'm a bit confused when I see `m_Strategy` in InlineContext vs. `m_inlineStrategy` in Compiler. Shall we just align with `m_InlineStrategy`?